manifests/package/debian.pp

```diff
@@ -33,17 +33,13 @@
 
     case $package_source {
       'nginx', 'nginx-stable': {
-        apt::source { 'nginx':
-          location => "http://nginx.org/packages/${distro}",
-          repos    => 'nginx',
-          key      => '573BFD6B3D8FBC641079A6ABABF5BD827BD9BF62',
+        apt::ppa { 'ppa:nginx/stable':
+          ensure => present,
         }
       }
       'nginx-mainline': {
-        apt::source { 'nginx':
-          location => "http://nginx.org/packages/mainline/${distro}",
-          repos    => 'nginx',
-          key      => '573BFD6B3D8FBC641079A6ABABF5BD827BD9BF62',
+        apt::ppa { 'ppa:nginx/stable':
```

Comment:
mainline is `ppa:nginx/development` according to https://launchpad.net/~nginx/+archive/ubuntu/development